lib/core/network/api/services/dao_service.dart

```diff
@@ -1,41 +1,38 @@
 import 'package:dio/dio.dart';
+import 'package:hypha_wallet/core/error_handler/model/hypha_error.dart';
+import 'package:hypha_wallet/core/logging/log_helper.dart';
 import 'package:hypha_wallet/core/network/api/services/remote_config_service.dart';
 import 'package:hypha_wallet/core/network/models/dao_data_model.dart';
 import 'package:hypha_wallet/core/network/networking_manager.dart';
+import 'package:hypha_wallet/ui/architecture/result/result.dart';
 
 class DaoService {
-  final NetworkingManager networkingManager;
-  final RemoteConfigService remoteConfigService;
+  final NetworkingManager _networkingManager;
+  final RemoteConfigService _remoteConfigService;
 
-  DaoService({required this.networkingManager, required this.remoteConfigService});
+  const DaoService(this._networkingManager, this._remoteConfigService);
 
-  Future<List<DaoData>> getDaos({
+  /// Get dao's for this account
+  Future<Result<List<DaoData>, HyphaError>> getDaos({
     required String accountName,
     required Network network,
   }) async {
     final String query =
         '{"query":"query profileDhos(\$username: String!, \$first: Int, \$offset: Int) { getMember(details_member_n: \$username) { docId __typename createdDate details_member_n memberofAggregate { count } memberof(first: \$first, offset: \$offset) { ... on Dao { docId details_daoName_n settings { settings_daoTitle_s settings_isHypha_i settings_logo_s settings_daoUrl_s } } } applicantof { ... on Dao { details_daoName_n settings { settings_daoTitle_s settings_daoUrl_s } } } } }","variables":{"username":"$accountName"}}';
 
-    final url = remoteConfigService.graphQLEndpoint(network: network);
+    final url = _remoteConfigService.graphQLEndpoint(network: network);
     try {
-      final res = await networkingManager.post(url, data: query);
+      final res = await _networkingManager.post(url, data: query);
       final Map<String, dynamic> response = res.data;
       final List<dynamic> memberOfList = response['data']['getMember']['memberof'];
       final List<DaoData> daos = memberOfList.map((member) {
         return DaoData.fromJson(member);
       }).toList();
-      return daos;
+      return Result.value(daos);
     } catch (error) {
-      print('Error accessing graphQL');
-      print(error);
-      if (error is DioError) {
-        final dioError = error;
-        print('message: ${dioError.message}');
-        print('status code: ${dioError.response?.statusCode}');
-        print('message: ${dioError.response?.statusMessage}');
-        print('dioError: $dioError');
-      }
-      rethrow;
+      LogHelper.e('Error accessing graphQL');
```

Comment:
pls use log helper and not print

Reply:
If your need to know is for debug purposes then we should not release with debug code. 
I added the loggers back tho